shared/logging_module.rb

```diff
@@ -14,16 +37,23 @@ def logger
             class_name = self.name.split("::").last
           end
 
+          progname ||= class_name
+
           thread_id = ""
 
-          if ENV["FASTLANE_CI_SUPER_VERBOSE"]
+          if ENV['FASTLANE_CI_SUPER_VERBOSE']
```

Comment:
We use `"` for all strings to keep it simple

Reply:
I neglected to run rubocop. :womp: